azurerm/azurerm_mariadb_server/simple/main.tf

```diff
@@ -0,0 +1,49 @@
+# Summary: A simple Azure MariaDB Server
+
+# Documentation: https://www.terraform.io/docs/language/settings/index.html
+terraform {
+  required_version = ">= 0.14.0"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 2.0"
+    }
+  }
+}
+
+# Documentation: https://www.terraform.io/docs/language/values/variables.html
+variable "azure_subscription_id" {
+  type = string
+}
+
+# Documentation: https://www.terraform.io/docs/language/providers/requirements.html
+# Documentation: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs
+provider "azurerm" {
+  features {}
+
+  subscription_id = var.azure_subscription_id
+}
+
+# Resource Group
+# Documentation: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/resource_group
+resource "azurerm_resource_group" "changeme_simple_mariadb_server_resource_group" {
+  name     = "changeme-simple-mariadb-database-resource-group"
+  location = "West Europe"
+}
+
+# MariaDB Server within the Resource Group
+# Documentation: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/mariadb_server
+resource "azurerm_mariadb_server" "changeme_simple_mariadb_server" {
+  name                = "changeme-simple-mariadb-database-name"
+  resource_group_name = azurerm_resource_group.changeme_simple_mariadb_server_resource_group.name
+  location            = azurerm_resource_group.changeme_simple_mariadb_server_resource_group.location
+
+  storage_mb = "5120"
+  sku_name   = "B_Gen5_1"
+  version    = "10.3"
+
+  administrator_login          = "changeme-admin-username"
```

Comment:
This has caused an apply error. See: https://github.com/ContainerSolutions/terraform-examples/runs/3220364029?check_suite_focus=true

```
/home/runner/work/terraform-examples/terraform-examples/bin/run_azurerm_examples.sh checking folder: azurerm/azurerm_mariadb_server/simple ... azurerm/azurerm_mariadb_server/simple
./run.sh/home/runner/work/terraform-examples/terraform-examples/azurerm/azurerm_mariadb_server/simple

...

azurerm_mariadb_server.changeme_simple_mariadb_server: Creating...
╷
│ Error: creating MariaDB Server "changeme-simple-mariadb-database-name" (Resource Group "changeme-simple-mariadb-database-resource-group"): mariadb.ServersClient#Create: Failure sending request: StatusCode=0 -- Original Error: Code="TokenTooLong" Message="The login that starts with 'changeme-admin-u' is too long. Maximum length is 16."
│ 
│   with azurerm_mariadb_server.changeme_simple_mariadb_server,
│   on main.tf line 36, in resource "azurerm_mariadb_server" "changeme_simple_mariadb_server":
│   36: resource "azurerm_mariadb_server" "changeme_simple_mariadb_server" ***
│ 
╵
/home/runner/work/terraform-examples/terraform-examples/azurerm/azurerm_mariadb_server/simple
```